ihmc-common-walking-control-modules/src/main/java/us/ihmc/commonWalkingControlModules/controlModules/foot/SwingState.java

```diff
@@ -658,7 +658,9 @@ private void fillAndInitializeBlendedTrajectories()
             adjustedWaypoint.changeFrame(footstepFrame);
             adjustedWaypoint.setReferenceFrame(adjustedFootstepFrame);
             adjustedWaypoint.changeFrame(worldFrame);
-            blendedSwingTrajectory.blendFinalConstraint(adjustedWaypoint, swingDuration, swingDuration - timeWhenAdjusted.getValue());
+            blendedSwingTrajectory.blendFinalConstraint(adjustedWaypoint, swingDuration, swingDuration);
+            swingTrajectorySmoother.updateErrorDynamicsAtTime(timeWhenAdjusted.getDoubleValue(), desiredPositionWhenAdjusted, desiredVelocityWhenAdjusted);
```

Comment:
The old way was trying to change the touchdown position way too quickly, which causes the feedforward acceleration to explode. This had been fixed with the smoother, but we hadn't incorproated the smoother in when there was a trajectory that was sent externally, like is the case here.